tools/Environments/DevHome.Environments/Helpers/DataExtractor.cs

```diff
@@ -33,7 +33,8 @@ public static List<OperationsViewModel> FillDotButtonOperations(ComputeSystem co
 
         if (supportedOperations.HasFlag(ComputeSystemOperations.Delete))
         {
-            operations.Add(new OperationsViewModel("Delete", "\uE74D", computeSystem.DeleteAsync, ComputeSystemOperations.Delete));
+            operations.Add(new OperationsViewModel(
+                _stringResource.GetLocalized("Operations_Delete"), "\uE74D", computeSystem.DeleteAsync, ComputeSystemOperations.Delete, windowEx));

```

Comment:
It looks like "Name" (the parameter you're passing "_stringResource.GetLocalized("Operations_Delete")" into) only gets used in GetHashCode(). If you're passing in ComputeSystemOperations.Delete, do you need to pass this in too, or will it always be "_stringResource.GetLocalized("Operations_Delete")" for that ComputeSystemOperations value?

Also, do you _want_ the Name in GetHashCode() to be localized?

Reply:
It is actually used in the UI, through 
`<customControls:CardFlyout ItemsViewModels="{x:Bind DotOperations}"/>`

This change is for the UI only.

Reply:
How will I get the value of WindowEx that I am passing?